code/game/machinery/computer/medical.dm

```diff
@@ -92,14 +85,10 @@
 /obj/machinery/computer/med_data/tgui_data(mob/user)
 	var/data[0]
 	data["temp"] = temp
-	data["scan"] = scan ? scan.name : null
-	data["authenticated"] = authenticated
-	data["rank"] = rank
 	data["screen"] = screen
 	data["printing"] = printing
-	data["isAI"] = isAI(user)
-	data["isRobot"] = isrobot(user)
-	if(authenticated)
+	tgui_login_data(data, user)
```

Comment:
Add a note here to say this will append to data, otherwise it just looks like its using data, not sending anything back.

Reply:
Sure